test/query/shorthand.test.ts

```diff
@@ -0,0 +1,176 @@
+import {AggregateOp} from 'vega-lite/src/aggregate';
+import {Channel} from 'vega-lite/src/channel';
+import {Mark} from 'vega-lite/src/mark';
+import {ScaleType} from 'vega-lite/src/scale';
+import {TimeUnit} from 'vega-lite/src/timeunit';
+import {Type} from 'vega-lite/src/type';
+
+import {SHORT_ENUM_SPEC} from '../../src/enumspec';
+import {spec as specShorthand, encoding, fieldDef as fieldDefShorthand} from '../../src/query/shorthand';
```

Comment:
@felixcodes  Hey you haven't fixed `encoding` to be consistent with `fieldDef` here!